src/experimental/tools/mediaCapabilitiesProber/probers/mediaContentType/index.ts

```diff
@@ -14,52 +14,44 @@
  * limitations under the License.
  */
 
-import { IMediaConfiguration } from "../../types";
+import { MediaSource_ } from "../../../../../compat";
+import {
+  IMediaConfiguration,
+  ProberStatus,
+} from "../../types";
 
 /**
+ * @param {Object} config
  * @returns {Promise}
  */
-function isContentTypeAPISupported(): Promise<void> {
```

Comment:
IMO that function only added cognitive burden.

Checking that the function exists before using it is also understood by TypeScript, which is a nice plus.